riscv/isa_parser.cc

```diff
@@ -260,6 +274,17 @@ isa_parser_t::isa_parser_t(const char* str, const char *priv)
     bad_isa_string(str, ("Zfinx/ZDinx/Zhinx{min} extensions conflict with Base 'F/D/Q/Zfh{min}' extensions"));
   }
 
+  if ((extension_table[EXT_ZCMP] || extension_table[EXT_ZCMT]) &&
+      (extension_table[EXT_ZCD] || (extension_table['C'] && extension_table['D']))) {
+    bad_isa_string(str, "'Zcmp/Zcmt' are incompatible with Zcd which is included "
+                        "when 'C' and 'D' extension are both present");
```

Comment:
"'Zcmp' and 'Zcmt' are incompatible with 'Zcd', and are incompatible with 'C' when 'D' is present"

Reply:
OK. I'll update this. By the way, the upper is the original description from the spec.

Reply:
Both are correct; this version is just a bit clearer.